doc/Proxy.md

```diff
@@ -0,0 +1,42 @@
+## Agora behind a reverse proxy (Incoming connections)
+
+Agora communicates other nodes through its configured `interfaces` which can be
+either HTTP or RPC (TCP) based. Some configurations might require putting an Agora
+node behind a reverse proxy. An Agora node utilizes ban manager to temporarily
```

Comment:
"However, and Agora node depends on a ban manager to [...]"

Since we're presenting the way two features conflict, the "however" serves to underline the contradiction.